src/v3/components/card_header/index.types.ts

```diff
@@ -0,0 +1,18 @@
+import { ReactNode } from 'react';
+export type CardHeaderPropsType = {
+  className?:
```

Comment:
@Logan08691: we already have a `CustomClassName` type in `@definition/app` if you would like to use it here.